src/Client.php

```diff
@@ -24,7 +25,7 @@
  *
  * @since   1.0
  */
-class Client implements HttpClient, HttpAsyncClient
+class Client implements HttpClient, HttpAsyncClient, ClientInterface
```

Comment:
Would it make sense to let `Http\Client\HttpClient` extend `Psr\Http\Client\ClientInterface`? Then, HTTPlug consumers would not have to update their type hints if they are currently using `HttpClient` instead of a concrete implementation (hopefully they do).

Reply:
It already does. :)

I just added it here to be more explicit.